lib/modules/manager/vendir/readme.md

```diff
@@ -0,0 +1,67 @@
+Renovate supports updating Helm Chart references in vendir.yml via the [vendir](https://carvel.dev/vendir/) tool. Renovate requires the presence of a [vendir lock file](https://carvel.dev/vendir/docs/v0.40.x/vendir-lock-spec/) which is generated by vendir and should be stored in source code.
+
+It supports both https and oci helm chart repositories.
+
+```yaml title="Example vendir.yml"
```

Comment:
ups, that's too verbose, only leave structure elements which are required for renovate.